docs/samples/Microsoft.ML.Samples/Dynamic/ValueMapping.cs

```diff
@@ -0,0 +1,281 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML.Data;
+using Microsoft.ML.Transforms;
+using Microsoft.ML.Transforms.Conversions;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public class ValueMappingExample
+    {
+        class SampleInfertDataWithFeatures
+        {
+            public float Age = 0;
+            public string Education = default;
+            public string EducationCategory = default;
+        }
+
+        class SampleInfertDataWithInducedCategory
+        {
+            public float Age = 0;
+            public float Induced = 0.0f;
+            public string InducedCategory = default;
+        }
+
+        class SampleInfertDataWithIntArray
+        {
+            public float Age = 0;
+            public string Education = default;
+            public int[] EducationCategory = default;
+        }
+
+
+        public static void ValueMappingTransform()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var ml = new MLContext();
+
+            // Get a small dataset as an IEnumerable.
+            IEnumerable<SamplesUtils.DatasetUtils.SampleInfertData> data = SamplesUtils.DatasetUtils.GetInfertData();
+            var trainData = ml.CreateStreamingDataView(data);
+
+            // Preview of the data.
+            //
+            // Age    Case  Education  induced     parity  pooled.stratum  row_num  ...
+            // 26.0   1.0   0-5yrs      1.0         6.0       3.0      1.0  ...
+            // 42.0   1.0   0-5yrs      1.0         1.0       1.0      2.0  ...
+            // 39.0   1.0   12+yrs      2.0         6.0       4.0      3.0  ...
+            // 34.0   1.0   0-5yrs      2.0         4.0       2.0      4.0  ...
+            // 35.0   1.0   6-11yrs     1.0         3.0       32.0     5.0  ...
+
+            StringToStringMappingExample(ml, trainData);
+            FloatToStringMappingExample(ml, trainData);
+            StringToKeyTypeMappingExample(ml, trainData);
+            StringToArrayMappingExample(ml, trainData);
+        }
+
+        ///<summary>
+        /// This example demonstrates the use of the ValueMappingEstimator by mapping string-to-string values. The ValueMappingEstimator uses
+        /// level of education as keys to a respective string label which is the value.
+        /// The mapping looks like the following:
+        /// <list>
+        ///     <item>0-5yrs -> Cat1</item>
```

Comment:
>item [](start = 17, length = 4)

will need to add a <description> tag after item, otherwise they don't render. so:

<item><description>0-5yrs -> Cat1</description></item>